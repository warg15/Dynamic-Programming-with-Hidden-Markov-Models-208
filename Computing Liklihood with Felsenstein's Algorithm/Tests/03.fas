>Annas-hummingbird
CACAACTGAGCTGTTGATATTTAGATTGACCCATATGATGTGTGGTTGTCTATCATCGTTGAAGCTATTCGCAATCTTGAGCGGACAAACGGCAGTTTGTTGACGCACCAGGAAGCTGTAACTGGATTGACGTTTCGAAGTATGTTGGCTCATGTATGTTCTCCGTATATAATAGGAGTCGTACGGAATCGTTGGGGCAC
>American-crow
CACAACTGGGTTGTGGATATTTAGATTGACCCATATGAAATATGGATATCCATCGTGATTGAATATATTCGCAATCTTAAACGGGTAAACGTTAGTTTGTTCATACATCTGGAAGTCGTATCTGGACTGACGTTGCGAAGTATGTTGGCACATGTATGATACCCGTATATAATAAGATGCGTCCGGAATTGTCGATGCAC
>Common-cuckoo
CACCAATGAGCTGTTGATATTTATATTGACCCATATAACAGGTGGATATCTATCGTCATTGAAGGTATCTGCAACCTTAAGCGGACAAACGTCAGTTTGTTGATACACCTGAAAGCTGTAACTGGTCTGACGTTTCGAAGTATGGTGGCACATGTATGATCACCGTATACCATACGAATGGTACGGAATCGTCGGGACAC
>Yellow-throated-sandgrouse
CACAACTGAGCTACTGATATTTAGATTGACCCATATGATATGTGGTTGCCTATCGTCATCAAAGCTATTCGCAATTTTAAACGGACAAACGTCAGTTTGTTGATTCACCTGGAAGCTTTAACTGGACTGACGTTCCGAAGTATGTTGGCACATGTATGATCCCCGTATATAATAGGAATCGCACGGAAGCGTTGGGGCAC
>Turkey
TAAGATTCAGCTGTTAGTAACTAGATTGACCAGTATGATGTGGGGTTGTCTATTAACATTGAAGCAATTCAAGATCTGAAACGGAGAAAAGTTAGTATTTTGATACGCCCAGAAGCTATGACTGCACTGACGTTCGGAAGTATGTTTGCAGATGTATGACCCCCGTATATAATAAGAATCGTACGAAAGCGTTGGGGCAC
>Turkey-vulture
CACAACTGAGCTGTTGATATTTAGCTTGACCCATATGATATGTGGTTGTCTATCGTCATTGAAGGTATCCGCAACCTTAAACGGACAAAGGTCAGTTTGTTGATACACCTGGAAGCTGTAACTGGACTGACGTTCCGAAGTATGTTGGCACATGTATGATCCCCGTATATAATAGGAATCGTACGGAATCGTTGGGGCAC
>Adelie-penguin
CACAACTGAGCTGTTCAGATTTAGATTGACCCATATGATATGTGGTTGTCTATCGTCATTGAAGCTACTCGCAACCTTTAACGGACAAACGTCAGTTTGTTGATACACCTGGAGGCTGTAACTGGACTGACGTTCCGAAGTATGTTCGCACATGTATGATCCCCGTATATAGTAGGAATCGTACGGAATCGTTGGGGCAC
>Northern-fulmar
GACAACTGAGCTGTTGATATTTAGATTGACCCATATGATATGTGGTTGTCTATCGTCATTGAAGCTATTCGCAACCTGTAACGGACAAACGTCAGTTTGTTGATACCCCTGGAGGCTGTAACTGGACTGACGTTCCGAAGTATGTTGGCACATGTATGATCCACGTATATAAAAGGAATCGTACGGAATCGTTGTGGCAT
>Bald-eagle
CACAATTGAGCTGCTGATATTTAAATTGACCCATATGATATGTGGATGTCTATCGTCATTGAAGCTATTCGCAACTTTAAATGGACAAACGTCAGTTTGTTGATACACCTGGAAGCTGTAACTGGATTGACGTTCCGAAGTATGTTGGCACATGTATGATCCCCGTATATAATAGGAATCGTACGGAATCGTTGAGGCAC
>Bar-tailed-trogon
CACGACTCAACTGTTGATATTTAGATTGACCCATATGATATGTGGTTGACTATCGACATTGAAGCTATTCGGAACCTTAAACGGACCAACGTCAGTTTGTTGATACACCTGGGAGCTGTATCTGTACCGACGTTCCGAAGTATGTTGACAAACGTATGACCCCTATATATAATAGGAATCGTACGGAATTGTTGGGGGAC
>Crested-ibis
CACAACTGAGCTGTTGATATTTAGATTGACCCATATGATATGTGGTTGTGGATCGTCATTGAAGCTATTCGCAACCTTCAACGGACAAACGTCAGTTTGTTGATACACCTGGAGGCTGTAACAAGACTGACGTTCCGAATTATGTTGGCACATGTATGATCCCCGTATATAATAGGAATCGTACGGAATCGTTGGGGTAC
>Dalmatian-pelican
CACAACCGAGCTGTTGATATTTAGATTGACCCATATGATATGTAGTTGTCCATCGTCATTGAAGCTATTCGCAATCCCTGACGGACAAACGTCAGTTTGTTGATACACCTGGAGGATGTAACTGGACTGACGTTCCGAAGTATGTTGGCACATGTATGATTCCCGTATATAATAGGAATCGTATGGAATCGTTGGGGCAC
>Brown-mesite
CACAACTGAACTGTTGACATTTAGATTGACCCATATGATATATGGTTGTCTATCGTCGTTGAAGCTATTCGCAACCTTAAATAGACAATCGTCAGTTTGTTCATACAGCTGGAAGCTGTAACTGGATTGACGTTCCGAAGTATGTTCGCACATCTATGATCTCCGTATGTATAAGGAACCGTAAGGAGGCGTTGGGGCAC
>Medium-ground-finch
CACAACTAAGCTGTGGATATTTAGATACACCCATATGAAATGTAGATGTCCATCGTGATTGAATATATTCGCAAGCTTAAACGGGTAAACGTTAGTTTGTTCATACATCTGGAAGTCTTGTCTGGACTGACGTTGCGAAGTATGGTGGCACGTGTATGATACCCGTATATAATAAGGTGCGTCCGGAATCGTCGGTGCAC
>Emperor-penguin
CACAACTGAGCTGTTCAGATTTAGATTGACCCATATGATATGTGGTTGTCTATCGTCATTGAAGCTATTCGCAACCTTTAACGGACAAACGTCAGTTTGTTGATACACCTGGAGGCTGTAACTGGACTGACGTTCCGAAGTATGTTGGCACATATATGATCCCCGTATATAGTAGGAATCGTACTGAATCGTTGGGGCAC
>Rhinoceros-hornbill
GACAACTGAGCTGTCCGTATTTAGATTGACCCACATGATATGCGGTTGTCTATCGTCATTGAAGCTAACCGGAACCTTAAACGGACAAACGTCAGCTTGTTGGTACACCTGGAAGCTATAGCTGGACTGACGTTCCGAAGTATTTTGGCACATGTATGATCCCCGTATATAATAGGAATCGTACAAAATCGTTGGAGCAG
>Grey-crowned-crane
CACAACTGAGCTACTGATATTTAGATTGACCCATATGATATGTGGTTGTCTATCGTCATTGAAGCTATTTGCAACCTTACACGGATAAGTGTCAGTTTGTTGATACACCTGGAAGCTGTAACTAGACTGACGTTCCAAAGTATGTTGGCACATGTAAGATCCCCGTATATAATAGGAGTCGTACGTAATCGTTGCGGCAC
>Downy-woodpecker
CAAAACTGAGCTGTTGATATTTGGATTGACCCATATGATATGTGGTTGTCTGTCACCATTGGAACTATGCGCAACTTTAAACGGACAAACGTCAGTTTGTTGACAGACCTGGAAGCTGTAACTGGGCTGACTTTCCAACTTATGTTGGCACGTGTAGGAGCCCCGTATATGGTAGGAATCTTACGGAATCGTTGGGTCAC
>Carmine-bee-eater
CGCAACTGAGCTGTTGATATTTTGATTGACCCGTATGATATGTGGTTGTCTATCGTCATTGAAGCCATTCGCAACCGTAAACGGACATACGTCAGTTTGTTGATAGACCTGGAAGCTGTAACTGGACTGACGTTCCGGAGTATGTTGACACATATATGATCCCTGTGTACAATAGGAATCGCACGGAATCGTTGGGGCAC
>Peregrine-falcon
CACAAATGAGCTGTTAATATTTAGATTGACTAATATGGTATGTGGTTGTCCATCGTCGTTGAAGCTATTCGCAACCTTAAACGGAAAAACGTCACTTTGTTGATACACCTGAAAGCTGTAACTAGACCGACGTTCCGAAGTATGTTGGCATGTGTATGATTCCCGTATATAATAGGAAGCGTAAGGAATCGTTGGGTCAC
>Common-ostrich
CACAATTGAACTGTAGATATTTAGATTGACCCGTATGATGCCTGGTTGTTCATCATCATCGAAGCTATTCGCAGCTTGAAACGAAGAAAAGGCAGTTTGTTGCTTCATATTGAAGCTTTAATTAGACCGACCCTCCGAAGTATGCTCGCACGCGTATGATCCCGGTATATAATAAGAATAGTACGGGAGCCTTGGGGCAC
>American-flamingo
CACAACTGAGCCGTTGATATTTAGGTTGACCCATATGATATGTGGTTGTCTATCGTCATTGAAGCTTTTCGCAACCTTAAACGGACAAACGTCAGTTTGTTGATACAGCTGAAAGCTGTAACCGGACTGACGTCCCGAAGTATGTTGGCACATGTATGATCCCCGTATATAATAGGAATCGTACGGAAGCGTTGGGGCAC
>Sunbittern
CACAACTGAGCTGTTGATATTTAGGTTGGCTTATATGATATGTGGTTGTCCATCGTCATTGAAGCTATTCGCAACGTTTGACGGACAAACGTCAGTTTGTGGATACACCTGGAAGCTGTAACTGGATTGACGTGCGGAAGTACGTTGGCAGATTTATGATCCCCGTATATAATAGGAATCGTACGGAATCGATGGGGCAT
>Speckled-mousebird
CCCAACTGAGCTGTTTATATTTATATTGACCCATATGATATGTGGTTGTTTATCATCATTGAAGCTATTCGGAACCTTTACCGGACAAACTTCAGTCTGTTGATGCACCTGGAAGCTGTAATTGGACTGACGTTCCGGCGTATGTTGGCACATGTATGATCCCCGTATACAATAGGAATCGAACGGAATTGTTGGGCCAC
>Chicken
TAAAATTGAGTTGTTAGTAACTAGATTGAACCGTATGATGTAGGGTTGTCTATTATCATTGAAGCAATTCAAGATCTGAAATGGAGAAAAGTTAGTATTTTGATACAACCAGAAGCTATAACTGCACTGACGTTCGGAAGTATATTTGCAAATGCATGACCCCCGTATATAATAAGAATCGTACGAAAGCGTTGGGGCAC
>Kea
TACAACTGAGCTGTTGATATCTAGATTGACCTGGATAATATGTGGTTTTCTATCGTCATTAAAGCTAATTGCAACCTAAAACGGACGAACGTCAGTTTGTTGATACACCTAGAAACTGCAACTGGATTGACGTGCCGCAGTATGTTGGTACATGTATGATCCCCGTACATAACAGGAAGCGTACGGAATCGTTGGGGTTC
>Red-legged-seriema
CACAACGGAGCTGTTGATATTTAGATTAACCTATATGGTATGCGGTTGCCTATCGTCATTCAAGCTATCCGTAACCTTAAACGGACAAACGTCAGTTTGTTGATACACCTGGAGGCTGTAACTGGACTGACGTTCCGAAGTATGTTGGCACATGTATGATCCCCGTATATAATAGGAAGCGTACGGCATCGTTGGGGCAC
>Barn-owl
CACAATTGAGCTGTTGATATTTAGATTGACCCATATCATATGTGGTTGTCTATCGTGATTGAAGCTGTTGGCAACCTTCAAAGGCCAAACGTCAGTTCCTTAATACACCTGGAAGCTGTAACTGGACTGACGTTCCGAAGTATGTTGGCACATGTATGATCCCCGTATATAATAGGAATAGTACGTAATCGTTGGTGCAC
>Budgerigar
CACAACTGAGCTGTTGATATTTAGATTGACCCATATGATATGTGGTTGTCTATAGTCATTAAAGCTATTTGCAACTTTAAACGGACAAACGTCAGTTTGTTGATACGCCTGGAAACTGCAACTGGATTGACGTTCCGCGGTATGTTGGTACATGTATGATCCCCGTATATAACAGGATGCGTATGGAATCGTTGGGGTAC
>Red-crested-turaco
CACAACTGAGCTGTTGATATTTAGATTGACCCATATGGTATGTGGCTGTCTATCATCATTGAAGCTGTTGGCAACCTTAAGCGGACAAACATCCGTTTGTTGATACAACTGGAAGCCGCAACTGGACTGACGTTCCGAAGTATGTTGGCACGTGTATGATCCCCGTATATAAGAGGAATCGTAGGGAATCGTTGGAGCAC
>Great-cormorant
CACAACTGAGCTGTTGACATTTAGATTGACCGATGTGCTATGTGGTTGTCTACCGTCATTGAAGCTATTCGCAACCTTTAACGGATAAACGTCAGTTTGTTGATACACCTGGAGGCTATAACTGGACTGACGTTCCGAAGTATGTTCGCTCATGTGTGATCCTCGTATATAATAGGAATAGTAGGGAATCTTTGGGGCAC
>Hoatzin
GAAAACTGAGCTGTCGATATTTAGATGGACCCATATGGTATGTGTTGGTCTGTCGTCATTGAAGCTATTCGCAACCTTAAACGGACAAGTGTCAGTTTGTTGATACACCTGGAAGTTGTAACTGGACTGACGATTCGAAGTATGTTGGCACATGTATAATCCCCGTATATAATAGGAATCGTACGGAATAATCGGGGCAC
>White-throated-tinamou
CACAATTGAGCTGTTGATATTTAGATTGACCCGTATGATACGTGCTTTTCTATCGTCATCGAAGGTATTCGCAACTTGAACCGAAGAGAAGTCTGTATGTTGATACATATGGGAGCTGTAACTAAACCGACCCTCCGAAATACGCTGGCGCACGTATGCTTTCGGGATATGATAAGAATAGTAGGAGAGCCTTCGGGCGC
>Pekin-duck
TACAGATAGGGTGTTGATATTAAGATTCACCTATATGATGTGGGGTTGTCTATCATCATTGAAGCAACTCAGAACATGAAATAGAGAACAGTCAGTTTGTTGATACACCTGGAAGCTGTAATTGGCCAGACGTTCCGAAGTATGTTTACAAATGTATGATCCCCGTATGTAATAAGAATTGAACGAAAACGTTGGGACAC
>MacQueens-bustard
CACAACTGAGCTGTTGATATTCAGATTGACTGATATGATATGTGGTTGTCTATCGTCATTGAAGCTATTCGCAACCTTAATCGGACAAACGCAACTTTGGTGATACACCTGGAAGCTGTAACTGGACTGAGGTTCCGAAGTATGTTGGCATATGTATGATCCCCGTACATAATAGGAGTCGTACGGAATCGTTGGGGCAC
>Killdeer
CACAACTGAGCTGTTGATATTTAGATTGACCCATATGATATGTGGCTGTCTATCGTAATTGAAGATATTAGCAACCCTAAACGGACAAGTGTCTGTTTGTTGATACACCTGGAAGCTGTAACTGGACTGACGTTCCGAAGTATATTGGCACATGTGTGATCCCCACATATAATAGGAATCGAACGGAATCGTTGGGGCAC
>Golden-collared-manakin
CACAACTGAGCAGTGGATATTTAGATTGACTCATACGCAATGTGGATGTCTATCGTGATTGAAGATATTCGCAAACTTAAACGTATAAACGACAATTTGTTAATACATCTGGAAGTTGTGACTGGACTGACGTTCCGACGTATGTTGGCACATATATGATACTCGTATATAATAGGATGCATGCGGAATCGTCGGGGCAC
>White-tailed-eagle
CACAATTGAGCTGCTGATATTTAAATTGACCCATATGATATGTGGATGTCTATCGTCATTGAAGCTATTCGCAACTTTAAATGGACGAACGTCAGTTTGTTGATACACCTGGAAGCTGTAACTGGATTGACGTTCCGAAGTATGTTGGCACATGTATGATCCCCGTATATAATAGGAATCGTACGGAATCGTTGAGGCAC
>White-tailed-tropicbird
CACAACTGACCTGTTGATATTTAAATTGACCCATATGATATGTGGTTGTCTATCGTCATTGACGCTATTCGCAACGTTTAACGGACAAACGTCAGTTTTTTGATATACTTGGAAGCTGTAACTGGACTGACGTTCCGAAGTATGTTGGCACATGGATGATCCCCGTATATAATAAGAATCGTACGGAATCGTTGGGGCAC
>Pigeon
CACGTCTGAGCTGTTGGTATCTAGATTGATCCGTATGATAGGTGGTCGTCTATCGTAATTGAATCTATTTACAAACTTAAACGGACGAACGTCAGTTTGTTTATACACCAGGTAGCTGGAACTGGACTGAGGTTCTGAAGTATGTTGGAACATGCATGATCCCCGTATATAATAGGAATCGTGCGGAAGTGTTGGGGCAC
>Chimney-swift
CACAACTGGGCTGTTGATATTTAGATTGACCCATACGATATGTGGTTGTCTATCATCATTAAAGCTATTTGCAACCTTAAACGGACAAACGTCAGTTTGTTGCTACACCTGGAAGCTGGAACTGGACTGACGTTCCAAAGTACGTTGGCACATCTATGATCCCCGTATATAACAGGAATCGTACGGAATCGTTTGGGCAC
>Red-throated-loon
CACAACTGGGCTGTTGATACCTAGATTGACCCATATGATATGTGATTGTCTATCGTCATAGGAGCTATTCGCAGACTTTAACGGACAAACGTCAGTTTGTTGATGCACCTGAAAGCTGTAACTGGACTGAAGTTCCGAAGTATTTTGGTACATGTATGATCCCCGTATATAATAAGAATCGTACGGAAGCGTTGGGGCAC
>Rifleman
CACAACTGAGCTGTGCATGTTTAGATTGAACCATATGAAATGTGGATGTATATCGTCCTTAAAGTTATTCGCAACCTTAAACGGATAAACGTCAGTTTGTTGATACATCTGGAAGTCGTAACAGGACTGACGTTCCGAAGTATGTTTGCACATGTATGATACCCGAATATAATATGATGGGTACGGAATCGTCGGGACAC
>Zebra-finch
CACAACTAGTCTGTGGATATTTAGATAAACCCATGTGAAATGTAGATGTCCATCTTGATTAAATATATTCGCAACCTTAAACGGGTGAACTTTAGTTTGTTCATACATCTGCAAGTCATGTCAGGACTGACGTTGCGAAATATGTTGGCACATGTATGATACCCGTATATAATAAGGTGCGTCCGCAATCGTCGGTGCAC
>Great-crested-grebe
CACAATTGAGCTGTTGATATTTAGATTGACCCATATGATATGTAGTTGTCTCTCGTCATTGAAGTTTTTCGCAACCTTAAACTGACAAACGTCAGTTTGTTGATACAGCTGGAAGCTGTAACTGGACTGACGTCCGGAAGTTTGTTGGAACATGTATGATCCCCGTATATAATAGGAATCGTATGGGAGCGCTGGGGCAC
>Cuckoo-roller
CACAACTAACCGGTTGATATTTAGATTGACCCATATGATATGTGGCTGTCTATCGTCATTGAAGCTATTCGCAACCCTAAACGGCCAAACGTCAGTTTGTTGATACACCCGGAAGCTGTAACTGGATTGACGTTTCGAAGTATGTTGGCAAATGTATGATCCCCTTATATAATAGGAATCGTACGAAATCGTTGGGGCAC
>Little-egret
CACAACGGAGCTGTTGATATTTAGATTGACCCATATGATATGTAGTTGTCTATCGTCATTGAAGCTATTCGCAACCTTTAACGGACAAACGTCAATTTGTTGTCACACCAGGAGGCTGTAACTGGACTAACGTTCCAAAGTATGTTGGCACATGTATGATCCCCGTATAAAGTAGGAATCGTGCGGAATCGTTGAGGCAC
>Chuck-wills-widow
CAAGACTGAGTTGTTGATATTTAGATTGACCCATGTGATATTTGGTTGTCTACCGTCATTGAAGCTATTCGCAGCCTTAAACGGACAAACATCAGTTTGTTGACACACCTGGAAGCTGTAACTGGTCTGACGCTCCAAAGTATGTTGGCACTTGTATGGTCCTCGTATATAATAGGAATCGTACGGAACCGTCGGGGCAC
