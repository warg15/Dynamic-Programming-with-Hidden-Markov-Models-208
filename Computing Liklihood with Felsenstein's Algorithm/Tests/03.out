-3526.8837952220215
